src/history/hs.c

```diff
@@ -698,7 +698,8 @@ __wt_hs_insert_updates(WT_SESSION_IMPL *session, WT_PAGE *page, WT_MULTI *multi)
     WT_MODIFY entries[MAX_REVERSE_MODIFY_NUM];
     WT_MODIFY_VECTOR modifies;
     WT_SAVE_UPD *list;
-    WT_UPDATE *first_non_ts_upd, *non_aborted_upd, *oldest_upd, *prev_upd, *upd;
+    WT_UPDATE *first_globally_visible_upd, *first_non_ts_upd, *non_aborted_upd, *oldest_upd,
+      *prev_upd, *upd;
```

Comment:
We don't wrap parameter lists like this. Add a new type declaration on a new line. Having enough local variables to need wrapping is an indication that the function is overloaded.

Reply:
Yes, I agree it is overloaded. I'll see if I can move some logic to another function.

Reply:
Created WT-6489 to refactor it.